### PR TITLE
Support wildcard licenses

### PR DESCRIPTION
**Pull request by LiaraAlis**

### CODE REVIEW

File: src/Components/Api/AccessToken.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public static function create(array $response): self
         $me = new self();
         $me->userId = (int) $response['userId'];
         $me->token = $response['token'];
-        $me->locale = $response['locale'];
+        $me->locale = $response['locale'] ?? 'de_DE'; // for sub-accounts the locale is missing in the response for some reason
```
Comment (shyim) [inline src/Components/Api/AccessToken.php:82]: please use en_GB

File: src/Components/Api/Client.php [hunk 2]
```diff
@@ -89,6 +90,27 @@ public function shops(AccessToken $token): array
             // no partner account
         }
 
+        // wildcard shops
+        $wildcardShops = [];
+
+        try {
+            $content = $this->client->request('GET', self::ENDPOINT . 'wildcardlicenses?companyId=15387&type=partner')->getContent();
```
Comment (shyim) [inline src/Components/Api/Client.php:97]: Remove your companyId 🙈

File: src/Components/Api/AccessToken.php (hunk 1 above)
Comment (LiaraAlis) [reply src/Components/Api/AccessToken.php]: Done. :)

File: src/Components/Api/Client.php (hunk 2 above)
Comment (LiaraAlis) [reply src/Components/Api/Client.php]: Oops! xD
Done!

**Pull request merged**